configs/sst-cores/a64fx-sst.yaml

```diff
@@ -28,6 +28,8 @@ Queue-Sizes:
   ROB: 128
   Load: 40
   Store: 24
+Port-Allocator:
+  Type: Balanced
```

Comment:
Also a64fx

Reply:
Updated (tested it works for all test binaries)

Reply:
This is still `Balanced` when it should be `A64FX` no?